# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/moz.build
# Commit: 6ea4f3bba40f
# Full Hash: 6ea4f3bba40f2fdf5c4893e0eb76fee7d4e05094
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2025-07-05 09:22:06
# Regressor Bug: 1912897
# File Overlap Count: 1
# Description:
#   Bug 1912897 - Support blurring and shadows for Accelerated Canvas2D. r=aosmond
#   
#   This adds support via FilterNodeWebgl for accelerating specific filters in AC2D.
#   In particular, it adds support for accelerated blurring using a separable blur
#   filter. This is then used to implement the Gaussian Blur filter as well as shadows.
# ==============================================================================

diff -r c6d00b59234d -r 6ea4f3bba40f dom/canvas/moz.build
--- a/dom/canvas/moz.build	Sat Jul 05 00:10:54 2025 +0000
+++ b/dom/canvas/moz.build	Sat Jul 05 04:41:59 2025 +0000
@@ -110,6 +110,7 @@
 
 SOURCES += [
     "DrawTargetWebgl.cpp",  # Isolate Skia
+    "FilterNodeWebgl.cpp",
     "ImageUtils.cpp",
     "OffscreenCanvasDisplayHelper.cpp",  # See bug 1745384
     "OffscreenCanvasRenderingContext2D.cpp",  # See bug 1745384